python/Ganga/Lib/Localhost/LocalHostExec.py.template

```diff
@@ -76,6 +76,8 @@ sys.path.insert(0, ###GANGADIR###)
 sys.path.insert(0,os.path.join(os.getcwd(),PYTHON_DIR))
 
 fullenvironment = os.environ.copy()
+for key,value in environment.iteritems():
+    fullenvironment[key] = value
 
```

Comment:
I don't quite follow this - why do you define `fullenvironment' and then assign values that look to be the same thing? Are those in environment different/additional?